### PR TITLE
Comparison table with Google ads

### PR DESCRIPTION
**Pull request by davidfischer**
Put the details of the landing page in table form. If this looks good, we could probably add a few more improvements.

## Screenshot

Screenshot from 2023-03-20 15-54-04

### CODE REVIEW

File: content/pages/vs-google.md [hunk 1]
```diff
@@ -1,15 +1,194 @@
...
+      </td>
+    </tr>
+    <!-- Cookies -->
+    <tr>
+      <td>
+        <div class="float-left p-3">
+          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" width="32" height="32"><!--! Font Awesome Pro 6.3.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2023 Fonticons, Inc. --><path d="M231.2 17c-22.1-3.1-44.6 .9-64.4 11.4l-74 39.5C73.1 78.4 57.2 94.9 47.4 115L10.7 190.6c-9.8 20.1-13 42.9-9.1 64.9l14.5 82.8c3.9 22.1 14.6 42.3 30.7 57.9l60.3 58.4c16.1 15.6 36.6 25.6 58.7 28.7l83 11.7c22.1 3.1 44.6-.9 64.4-11.4l74-39.5c19.7-10.5 35.6-27 45.4-47.2l36.7-75.5c9.8-20.1 13-42.9 9.1-64.9l-14.6-82.8c-3.9-22.1-14.6-42.3-30.7-57.9L372.9 57.5c-16.1-15.6-36.6-25.6-58.7-28.7L231.2 17zM192 144a32 32 0 1 1 0 64 32 32 0 1 1 0-64zM128 336a32 32 0 1 1 64 0 32 32 0 1 1 -64 0zm224-64a32 32 0 1 1 0 64 32 32 0 1 1 0-64z"/></svg>
+        </div>
+        <div>
+          <strong>Cookies &amp; cookie banners</strong><br>
```
Comment (ericholscher) [inline content/pages/vs-google.md:54]: Seems we want to make this a positive? We're putting a check here on `Cookies and cookie banners`, where I think we want `No cookies and privacy regulation (GDPR) compliant` or something that's framed positively? 

The red check mark is really confusing to me also. I think we want an ❌ ?

File: content/pages/vs-google.md [hunk 2]
```diff
@@ -18,12 +197,16 @@ When your user goes to your website, they are assuming they will only be sending
 
 When you use EthicalAds, **we never set cookies on your users**, and we don't use any user-specific data to target advertising. This means [you don't need a cookie banner]({filename}../posts/2021-can-you-remove-cookie-banners.md) when you use our ads, and your users data is not being tracked across the internet.
 
+
+<a id="transparency"></a>
```
Comment (ericholscher) [inline content/pages/vs-google.md:201]: I wish pelican just made each heading linkable by default without needing to enable the TOC extension.

**Review by ericholscher — COMMENTED** [review]
This is a useful start, but I think we want the red items to be X's instead of ✔️ 's? 

There's definitely some additional work to do here on the framing, but mostly just noting the visual feedback.

File: content/pages/vs-google.md (hunk 1 above)
Comment (davidfischer) [reply content/pages/vs-google.md:54]: We definitely want an X.

**Review by ericholscher — APPROVED** [review]
This is a solid improvement 👍 The icons look a little washed out to me when at default zoom. They actually show more clearly when I zoom in -- not really sure what's up with that, but thought I'd note it. 

Screenshot 2023-03-22 at 11 14 59 AM

**Comment by davidfischer** [comment]
> This is a solid improvement  The icons look a little washed out to me when at default zoom.

Odd. They look completely crisp for me in both Chrome and FF.

**Pull request merged**